### ISSUE TITLE
Jobs will not run for non reasonable requirements

### ISSUE THREAD

**Issue by juztas** [dmwm/WMCore#6108]
There are some number of jobs from time to time, which requests not reasonable memory/walltime requirement. Due to this jobs are sitting idle and not running. For example this workflow `cerminar_Run2015B-L1Accept-75XPreProd-22Jul2015_751_150805_235531_8107 ` requests(Object 1 - idle, 2 - running).
```
Mem||Wall||Desired_Sites
2500||20000||T1_US_FNAL
	Object { 1=72,  2=0}
2500||3000||T1_US_FNAL
	Object { 1=14,  2=0}
2500||17500||T1_US_FNAL
	Object { 1=454,  2=0}
2500||27500||T1_US_FNAL
	Object { 1=99,  2=0}
2500||10000||T1_US_FNAL
	Object { 1=26,  2=0}
2500||22500||T1_US_FNAL
	Object { 1=62,  2=0}
```
And entries for site, how much is it providing: (Num - entry number(equal entries are combined))
```
Num	MaxWallTime	MaxMemory
0	2850	24576
1	2850	20000
```
So none of these jobs will run. System could be more intelligent and not specify walltime (22k minutes which is 15days)
What is the longest job which was running? (Can be identified in dashboard)
Limit all jobs to max 47.5h walltime request.

There is also issues with memory requests (requesting 3GB from T2_CH_CERN while it provides max 2.8GB). For this I am implementing for crab recurring action, which gets all information from factory entries and in case user ask too high memory/walltime, crab will give a warning that these jobs will not run and you have to edit your requirements.

I have reported more problematic workflows like this on ELOG.

**Comment by hufnagel** [comment]
The EventAwareLumiBased splitting supports setting a cap for wall time and disk, but when I pointed this out the reply was that in production the timePerEvent and sizePerEvent limits are reliable enough so this isn't needed.

Basically what the cap does is to use the estimates based on the algorithm if its below the cap (so we trust the algorithm there). If it's above we set it to the cap though to ensure that the job will run (with maximum settings that get the jobs still to run).

IMO, no cap makes no sense unless we also have a secondary mechanism that makes sure these jobs get automatically killed/failed. Submitting a job to HTCondor that will never run makes no sense.
